test/object_detector_filter_test.jl

```diff
@@ -0,0 +1,63 @@
+image_with_objects = zeros((30,30));
```

Comment:
Please, do not share global variable between test functions, Just add them in the functions